samples/js-tests/src/CocoStudioTest/GUITest/UIBaseLayer.js

```diff
@@ -49,6 +49,18 @@ var UIBaseLayer = cc.Layer.extend({
         this._mainNode = mainNode;
         this._topDisplayText = topDisplayText;
     },
+
+    _parseUiFile: function(file){
+        if(cocoStudioTestScene_api == 0){
```

Comment:
Variable name should be changed, it's not comprehensive. I don't understand what it means